### PR TITLE
Add back symbols to TASTy reflect

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: compiler/src/dotty/tools/dotc/tastyreflect/TastyImpl.scala [hunk 1]
```diff
@@ -2,4 +2,4 @@ package dotty.tools.dotc.tastyreflect
 
 import dotty.tools.dotc.core._
```
Comment (liufengyun) [inline compiler/src/dotty/tools/dotc/tastyreflect/TastyImpl.scala]: Format the following line into several lines?

File: library/src/scala/tasty/Tasty.scala [hunk 2]
```diff
@@ -2,4 +2,4 @@ package scala.tasty
 
 import scala.tasty.reflect._
```
Comment (liufengyun) [inline library/src/scala/tasty/Tasty.scala]: Reformat the following line to be shorter?

File: library/src/scala/tasty/reflect/SymbolOps.scala [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package scala.tasty
+package reflect
+
+/** Tasty reflect symbol */
+trait SymbolOps extends TastyCore {
+
+  trait SymbolasAPI {
```
Comment (liufengyun) [inline library/src/scala/tasty/reflect/SymbolOps.scala]: `SymbolasAPI ` -> `SymbolsAPI `

File: library/src/scala/tasty/reflect/SymbolOps.scala [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    def isEmpty: Boolean
+    def localContext(implicit ctx: Context): Context
+    def tree(implicit ctx: Context): Option[Definition]
+  }
+  implicit def SymbolasDeco(symbol: Symbol): SymbolasAPI
```
Comment (liufengyun) [inline library/src/scala/tasty/reflect/SymbolOps.scala]: The same as above.

File: compiler/src/dotty/tools/dotc/tastyreflect/TastyImpl.scala (hunk 1 above)
Comment (nicolasstucki) [reply compiler/src/dotty/tools/dotc/tastyreflect/TastyImpl.scala]: Done

File: library/src/scala/tasty/Tasty.scala (hunk 2 above)
Comment (nicolasstucki) [reply library/src/scala/tasty/Tasty.scala]: Done

**Pull request merged**